utils/tests/test_wait.py

```diff
@@ -25,7 +25,7 @@ def test_simple_wait():
     ec, tc = wait_for(incman.i_sleep_a_lot,
                       fail_condition=0,
                       delay=.05)
-    print "Function output %s in time %s " % (ec, tc)
+    print("Function output %s in time %s " % (ec, tc))
```

Comment:
while you modified the formatters in the print statements, you can also do it in this file, this looks like it was forgotten :)